### PR TITLE
Make module formatting not generate error with strict mode

### LINKED ISSUE #11909: Get-Module is polluting $error (from formatting script) (when strict mode on)
Seems this error comes from PowerShellCore_format_ps1xml.cs, from a handful of script blocks that look like this:

```csharp
.AddScriptBlockColumn(@"
    if ($_.PrivateData -and $_.PrivateData.PSData)
    {
            $_.PrivateData.PSData.PreRelease
    }")
```

And of course `PreRelease` will not be defined for many (most?) modules that are out there.

# Steps to reproduce

```powershell
$error.Clear()
Set-StrictMode -Version Latest
New-ModuleManifest .\foo.psd1
ipmo .\foo.psd1
Get-Module foo | Out-Null
$error
```

# Expected behavior

There should be no errors in `$error`, so there should be no output from the repro steps.

```none
<no output>
```

# Actual behavior
Error!

```none
PropertyNotFoundException:
Line |
   4 |                                      $_.PrivateData.PSData.PreRelease
     |                                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | The property 'PreRelease' cannot be found on this object. Verify that the property exists.
```

Although it doesn't show up in normal output, if you look into`$error`, it's there, making you wonder what went wrong. If you are looking into `$error`, you are probably trying to troubleshoot some problem, and you only want to see "real" errors in `$error`, and not be distracted or confused by things that just look like bugs in PowerShell.

Looking at it another way: I didn't run some command with `-ErrorAction SilentlyContinue`, and I didn't see an error from the command... so why in the world is an error showing up in `$error`? (That's why I described it as "polluting `$error`"--it seemingly is not an error; just the result of some internal PS bug scribbling useless and distracting information into it.)

# Environment data

```none
Name                           Value
----                           -----
PSVersion                      7.0.0-preview.7
PSEdition                      Core
GitCommitId                    7.0.0-preview.7
OS                             Microsoft Windows 10.0.19570
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### PR DESCRIPTION
**Pull request by jazzdelightsme**
Fixes #11909

# PR Summary

Minor tweak to module formatting script to avoid errors when strict mode is on.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

File: src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs [hunk 1]
```diff
@@ -1632,6 +1632,14 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_Microsoft_PowerShell_Co
                 .EndTable());
         }
 
+        private const string GetPreReleaseStringScriptBlock = @"
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs]: The `Get` prefix makes it sound like this is a function.  I think just calling it `PreReleaseStringScriptBlock` is better.

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs (hunk 1 above)
Comment (jazzdelightsme) [reply src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs]: done!

---
In reply to: [385355079](https://github.com/PowerShell/PowerShell/pull/11943#discussion_r385355079) [](ancestors = 385355079)

**Comment by jazzdelightsme** [comment]
@SteveL-MSFT Quickest way to get me using a pre-release build would be to ship this change in one. ;)

**Review by vexx32 — APPROVED** [review]

**Comment by TravisEz13** [comment]
@SteveL-MSFT  Can you update your review?

**Comment by TravisEz13** [comment]
pinged @SteveL-MSFT  offline to update his review

**Review by SteveL-MSFT — APPROVED** [review]
Sorry for the delay!

**Pull request merged**